File: probes/python/loadavg.py
import os
import subprocess

class LoadAvg(Probe):
    def run(self):
        load1, load5, load15 = os.getloadavg()
        
        # Python 2/3 compatibility implementation
        rawCpuInfo = open('/proc/cpuinfo').read().splitlines()
        cpuCount = 0
        for line in rawCpuInfo:
            if line.startswith("processor\t:"):
                cpuCount += 1

        self.sendResults({'1m': load1, '5m': load5, '15m': load15, 'nproc': cpuCount})

LoadAvg().run()

File: probes/php/loadavg.php
<?php

$load = array_slice(preg_split('/\s+/', file_get_contents('/proc/loadavg')), 0, 3);
$nproc = exec('nproc');

$probe = new Probe();
$probe->sendResults(array(
    '1m' => (float) $load[0],
    '5m' => (float) $load[1],
    '15m' => (float) $load[2],
    'nproc' => (int) $nproc
));